src/bigquery.ts

```diff
@@ -878,23 +892,7 @@ export class BigQuery extends common.Service {
    * A geography value represents a surface area on the Earth
    * in Well-known Text (WKT) format.
    *
-   * @method BigQuery.geography
```

Comment:
I assume this was removed because it's documented at the actual method definition now?

Reply:
Yes, it's redundant with TSDoc.